etc/docker.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/bash
+
+# script to aid in local testing of linux platforms
```

Comment:
I'm happy to remove this, but I find this immensely useful when working on ubuntu or alpine builds locally.  We have this script in our zstd repo.